### PR TITLE
Fix-up NUnit 2 & 3 use in Unix Makefile

### PR DESCRIPTION
**Pull request by atlimit8**
Cleans up NUint 2 & 3 after #10309 and #10327.

### CODE REVIEW

**Comment by obrakmann** [comment]
I think it was the point to keep the possibility of still using nunit 2.

File: Makefile [hunk 1]
```diff
@@ -229,18 +229,20 @@ check: utility mods
 	@mono --debug OpenRA.Utility.exe ra --check-code-style OpenRA.Test
 
 NUNIT_CONSOLE := $(shell test -f thirdparty/download/nunit3-console.exe && echo mono thirdparty/download/nunit3-console.exe || \
-	which nunit2-console 2>/dev/null || which nunit3-console 2>/dev/null || which nunit-console 2>/dev/null)
```
Comment (atlimit8) [inline Makefile]: Even if NUnit 2.6 support is kept, these two need to be swapped:
```
which nunit3-console 2>/dev/null || which nunit2-console 2>/dev/null || which nunit-console 2>/dev/null)
```

**Comment by atlimit8** [comment]
Also if NUnit 2.6 support is kept, then the changelog should say something like: "Changed the preferred NUnit version to 3.0.1 (from 2.6.4)" instead of "Changed the used NUnit version to 3.0.1 (from 2.6.4)"

**Comment by atlimit8** [comment]
Changed to "Fix-up NUint 2 & 3 use in Unix Makefile" instead.

**Comment by Phrohdoh** [comment]
LGTM :+1:

**Comment by atlimit8** [comment]
Fixed a couple of typos as well.

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/d15a54ddb88b9119dc4fa16f3ce7ab0dc205b9e6...549a6e1719c582994ae4a79eef6109053142a371)

**Pull request merged**